airbyte-webapp/src/components/ui/BarChart/BarChart.tsx

```diff
@@ -63,7 +64,12 @@ export const BarChart: React.FC<BarChartProps> = React.memo(({ data, legendLabel
         >
           <Label value={yLabel} fontSize={11} fill={chartTicksColor} fontWeight={600} position="top" offset={10} />
         </YAxis>
-        <Tooltip cursor={{ fill: chartHoverFill }} />
+        <Tooltip
+          cursor={{ fill: chartHoverFill }}
+          formatter={(value: number) => {
+            return [<FormattedNumber value={value} />, yLabel];
```

Comment:
This is non-intuitive.  Rechart renders this as:
`${yLabel}: ${value}`

They need to be returned in this order (`[value, name]`).  The separator is configurable (and we can skip the whole thing for a custom Tooltip if we so desire)